addons/account/static/tests/reconciliation_tests.js

```diff
@@ -649,8 +649,8 @@ QUnit.module('account', {
         assert.strictEqual(widget.$('.o_input_dropdown input').val(), "Agrolait", "the partner many2one should display agrolait");
         assert.strictEqual(clientAction.widgets[2].$('.o_input_dropdown input').val(), "Camptocamp", "the partner many2one should display Camptocamp");
         widget.$('.accounting_view tfoot td:first').trigger('click');
-        assert.strictEqual(widget.$('.create input.o_input').length, 7,
-            "create panel should contain 7 fields (account_id, tax_id, journal_id, analytic_account_id, analytic_tag_ids, label, amount)");
+        assert.strictEqual(widget.$('.create input.o_input').length, 8,
+            "create panel should contain 7 fields (account_id, tax_id, journal_id, analytic_account_id, analytic_tag_ids, label, amount, date)");
```

Comment:
I would have added a business test to make sure the date is sent to the server

Reply:
and you still can.... :-) 
BTW, I just realize the message raised still says '7 fields' -_-

Reply:
That's weird, I ran the tests and I had to revert some previously changed tests, but this one passed without any problem.

Reply:
@william-andre what? no one is saying the test is broken.. I just say that, if it fails, it will say 
> create panel should contain **7** fields

although we're expecting and testing for 8 fields

Reply:
I didn't say it is broken either, only that it passed without failing on my local server